### DISCUSSION TITLE
More images

### DISCUSSION THREAD

**Discussion by janstehno** [janstehno/cotw-companion#7] [Ideas]
I do not think this is an important addition but it was suggested a few times. It would include addition of weapon's and caller's images (would look like animal heads already added to the app). There was also a suggestion to add furs in images (although this would take me some time if I would like to have it consistent and same for every animal).

**Comment by Khhunt** [comment]
Since the search for animal fur was added in the new update, I suggest that they could also have pictures of what it looks like on an animal. I think this would be a very useful addition.

**Comment by janstehno** [comment]
So I was able to **data-mine** something like this for furs. It's just an in-game model with a raw texture placed on it. For example, **melanistic** fur looks a bit weird compared to the one in the game. It is then rendered with some lights around. I have to keep it in a **smaller resolution** so it does not make the app much bigger. But I think this could suffice for now.

<img src="https://github.com/user-attachments/assets/5051f21d-3923-4a36-9e67-59849c841f51" alt="axis_deer_albino" width="280">
<img src="https://github.com/user-attachments/assets/a1bfe65e-7367-4496-b760-d3e6540ae5e1" alt="axis_deer_dark" width="280">
<img src="https://github.com/user-attachments/assets/dfdbbe6b-0bff-4510-84c7-e8e8fe33d067" alt="axis_deer_melanistic" width="280">
<img src="https://github.com/user-attachments/assets/4b3f4b31-f369-4f88-9900-5956371c497a" alt="axis_deer_orange" width="280">
<img src="https://github.com/user-attachments/assets/bcbf96e8-652b-49c3-a416-c844e8612e4f" alt="axis_deer_piebald" width="280">
<img src="https://github.com/user-attachments/assets/2f22fcba-a033-4bca-96e5-3204ec7c4a81" alt="axis_deer_piebald_female" width="280">
<img src="https://github.com/user-attachments/assets/f96e4cda-3654-4c8f-a590-b63a14201d19" alt="axis_deer_spotted" width="280">
<br></br>

I could also make full 3D 360° models. The problem is that they are so big I would have to load them from the internet rather than include them inside the app. In comparison, the model for one animal (due to high-res textures) has around **15 - 20MB**. The rendered images above have around **0.5 - 1MB** together.

**Reply by janstehno** [reply]
So far, I just had to place textures on models. I used and rendered models with no fur to make them more realistic. However, I stumbled upon Red Fox where the base model for it looks pretty weird if I render it the same way the Axis Deer is rendered in the previous post. So I was forced to add some fur to make it look better. The problem is, I can't exactly replicate the look from the game. Below are rendered images of Red Fox without and with my manually created fur. The Fox still looks **moutled** in my humble opinion but I hope it won't be a problem. I tried my best. There is a possibility that more animals will have to get this additional fur.

<img src="https://github.com/user-attachments/assets/99f11c22-3d8e-4cdc-b859-14c64df97792" alt="red_fox_red_without_fur" width="280">
<img src="https://github.com/user-attachments/assets/cf2e6125-d59d-40bc-9905-1e60e5a2fe7f" alt="red_fox_red_with_fur" width="280">

**Reply by janstehno** [reply]
Added in the version 1.10.11
